handlers/public.go

```diff
@@ -167,3 +170,10 @@ func getPaymentResource(w http.ResponseWriter, req *http.Request, resource strin
 	}
 	return paymentResource, nil
 }
+
+// Generates a string of 20 numbers made up of 7 random numbers, followed by 13 numbers derived from the current time
+func generateID() (i string) {
+	ranNumber := strconv.Itoa(1000000 + rand.Intn(900000))
```

Comment:
Won't this always begin with 1? The first digit should be random too.
Perhaps `strconv.Itoa(rand.Int(9999999))`?